src/storage/sftp/storage.c

```diff
@@ -831,31 +832,52 @@ storageSftpNew(const String *const path, const String *const host, const unsigne
             THROW_FMT(ServiceError, "libssh2 handshake failed");
 
         int hashType = LIBSSH2_HOSTKEY_HASH_SHA1;
+        size_t hashSize = 0;
```

Comment:
Would be good to have a comment here mentioning that if other hash types are later added, one should consider if the fingerprint string size needs to be adjusted below.

Reply:
Addressed in d080b7f9071478ed0730f5ee15efe0b4cfba13b4